api/metrics.go

```diff
@@ -21,7 +21,9 @@ import (
 var (
 	metricHTTPReqCounter       = metrics.LazyLoadCounterVec("api_request_count", []string{"name", "code", "method"})
 	metricHTTPReqDuration      = metrics.LazyLoadHistogramVec("api_duration_ms", []string{"name", "code", "method"}, metrics.BucketHTTPReqs)
-	metricActiveWebsocketCount = metrics.LazyLoadGaugeVec("api_active_websocket_count", []string{"subject"})
+	metricWebsocketDuration    = metrics.LazyLoadHistogramVec("api_websocket_duration", []string{"name", "code"}, metrics.BucketHTTPReqs)
```

Comment:
The bucket might require a unique one, most wescoket will have a long duration. And is there somewhere to describe this to be in seconds?

Reply:
So the current bucket is:

```golang
        BucketHTTPReqs = []int64{
		0, 1, 2, 5, 10, 20, 30, 50, 75, 100,
		150, 200, 300, 400, 500, 750, 1000,
		1500, 2000, 3000, 4000, 5000, 10000,
	}
```

Since we record in seconds, 10000 -> 166 minutes. Anything over this will be recorded in the `Inf` bucket. What timeframe would you suggest to add?

Reply:
Could go up to days/ weeks /months I guess

Reply:
> Since we record in seconds

Reply:
Updated!